### ISSUE TITLE
Add support for cookbook context

### ISSUE THREAD

**Issue by cknowles** [rackspace-cookbooks/chef-slack_handler#17]
It'd be nice to see the cookbooks and versions that have run in the notification to Slack. Would just need to incorporate something like `#{run_status_cookbooks}` into the `slack_message` calls then define this method:

```
  def run_status_cookbooks
    cookbooks = run_status.run_context.cookbook_collection
    "#{cookbooks.keys.map {|x| cookbooks[x].name.to_s + ' ' + cookbooks[x].version} }"
  end
```

I mostly grabbed that [from here](https://docs.chef.io/handlers.html) and yet to try a run of this.

**Comment by cknowles** [comment]
Actually what I was looking for here was just the cookbook being run so I ran this and it's a bit too verbose. Just working on a pull request to add the cookbook being run.

**Comment by martinb3** [comment]
I think this is probably fixed by #19 as well.
